.travis.yml

```diff
@@ -73,6 +73,19 @@ jobs:
       stage: Tests - phase 1
       script: ${MVN} animal-sniffer:check --fail-at-end
 
+    # Experimental run of the revised ITs. Done early to debug issues
+    # Move later once things work.
+    - name: "experimental docker tests"
```

Comment:
Do these have to be moved to a later stage before we can merge this PR?

Reply:
I went ahead and disabled this step for now: it has done its task of proving that the new ITs work in Travis. We'll revisit as we decide how to migrate from the existing IT groups to the new ones.